templates/home.html

```diff
@@ -30,6 +30,20 @@
             bottom: 0;
             width: 100%;
         }
+        @media only screen and (max-width: 900px) {
+            .d-flex {
```

Comment:
Hi @Kuldeepburjbhalaike, seems, you are creating new class .d-flex style, but not used anywhere. I am not sure how it will be applied on HTML element.

Reply:
hi @Jayprakash-SE, the class **.d-flex** is used in footer>div.